From 1d1ce8af1736e1968e83379b992c05b37a22220b Mon Sep 17 00:00:00 2001
From: Gatien Chevallier <gatien.chevallier@foss.st.com>
Date: Wed, 5 Jun 2024 13:36:46 +0200
Subject: [PATCH 0898/1695] core: lib: scmi-server: drop CFG_CROSS_COMPILE when
 compiling SCP firmware

No need to specify this build directive as there is no cross-compilation
using CMake. This solve the following warning:

CMake Warning:
  Manually-specified variables were not used by the project:

    CFG_CROSS_COMPILE

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git d110ad76d054d840554f473853ec8588aca1c982 core: lib: scmi-server: drop CFG_CROSS_COMPILE when compiling SCP firmware]
Change-Id: I2b6a2d69411cc66ef1dbdc5821aadb70634dfb37
Signed-off-by: Gatien Chevallier <gatien.chevallier@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/385401
Reviewed-by: Etienne CARRIERE <etienne.carriere@foss.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Tested-by: Gatien CHEVALLIER <gatien.chevallier@st.com>
Domain-Review: Gatien CHEVALLIER <gatien.chevallier@st.com>
Reviewed-by: Gatien CHEVALLIER <gatien.chevallier@st.com>
---
 core/lib/scmi-server/sub.mk | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/core/lib/scmi-server/sub.mk b/core/lib/scmi-server/sub.mk
index f321c12f9..695a5419c 100644
--- a/core/lib/scmi-server/sub.mk
+++ b/core/lib/scmi-server/sub.mk
@@ -35,8 +35,7 @@ scpfw-cmake-flags-y = -DSCP_FIRMWARE_SOURCE_DIR:PATH=$(scpfw-product)/fw \
 		      -DSCP_LOG_LEVEL="TRACE" \
 		      -DDISABLE_CPPCHECK=1 \
 		      -DCFG_NUM_THREADS=$(CFG_NUM_THREADS) \
-		      -DSCP_OPTEE_DIR:PATH=$(CURDIR) \
-		      -DCFG_CROSS_COMPILE=$(lastword $(CROSS_COMPILE_core))
+		      -DSCP_OPTEE_DIR:PATH=$(CURDIR)
 
 # CMake does not need to check the cross compilation toolchain since we do not
 # compile any source file with CMake, we only generate some SCP-firmware
-- 
2.47.2

